src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,35 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Stores type of task with deadline.
+ */
+public class Deadline extends Task {
+
+    protected LocalDate by;
```

Comment:
Perhaps the instance variable name can be more descriptive, e.g. `deadlineDate` or `date`?